File: chap11/run-model11-3.R
library(cmdstanr)

d <- read.csv('input/data-weight.csv')
T <- nrow(d)
Tp <- 3
data <- list(T=T, Tp=Tp, Y=d$Y)

model <- cmdstan_model('model/model11-3.stan')
fit <- model$sample(data=data, seed=123, parallel_chains=4)
fit$save_object(file='output/result-model11-3.RDS')

File: chap11/run-model11-3.py
import pandas
import cmdstanpy

d = pandas.read_csv('input/data-weight.csv')
T = len(d)
Tp = 3
data = {'T':T, 'Tp':Tp, 'Y':d.Y}

model = cmdstanpy.CmdStanModel(stan_file='model/model11-3.stan')
fit = model.sample(data=data, seed=123, parallel_chains=4)
fit.save_csvfiles('output/result-model11-3')
